scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -342,7 +343,6 @@
                                     (fn [now]
                                       (let [host->combined-offers
```

Comment:
Which offers is this seeing? Pre or post fenzo? If pre, change the name since we are no longer combining offers here.